core/server/master/src/main/java/alluxio/master/file/DefaultFileSystemMaster.java

```diff
@@ -2980,6 +3110,22 @@ private void setAttributeFromEntry(SetAttributeEntry entry)
     return mBlockMaster.getWorkerInfoList();
   }
 
+  private void checkAndAppendAuditLog(RpcUtils.RpcContext context) {
+    if (!mAuditLog.isEnabled()) { return; }
+    mAuditLog.append(context);
+  }
+
+  private void checkAndCommitAuditLog(String command, String src, String dst, Inode inode, RpcUtils.RpcContext context) throws AccessControlException {
+    if (!mAuditLog.isEnabled()) { return; }
+    context.setCommand(command).setUser(AuthenticatedClientUser.getClientUser())
```

Comment:
Does Alluxio employ a lazy way to set the client user? What it means when the current client user is not set? @ifcharming